rpi3/firmware/optee

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
```

Comment:
Yes that's what I've done. I have no strong preference to autostart it or just do it manually. For development I prefer doing it manually, since you have the ability to run gdb, valgrind etc "from start" on the binary.